statistics/services.py

```diff
@@ -0,0 +1,16 @@
+from creation.models import TutorialResource, Language
+
+LANGUAGES = Language.objects.all()
+LANGUAGES_IDS = LANGUAGES.values_list('id', flat=True)
```

Comment:
Don't do this it will return wrong result when deployed as it will be executed only once during module loading

Reply:
This is not being used in this logic.
In Bidding module it is used, but I didnt like it , so I changed it anyways.